pkg/maps/cells.go

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package maps
+
+import (
+	"github.com/cilium/cilium/pkg/hive/cell"
+	"github.com/cilium/cilium/pkg/maps/ipcache"
+)
+
+var Cell = cell.Module("maps", "Maps",
```

Comment:
Could expand the title a bit to include that it's wrappers for managing Cilium's BPF maps and add some more comments about what the purpose of this grouping is and what is provided.